source/read/cursor.txt

```diff
@@ -0,0 +1,179 @@
+.. _php-cursors:
+
+=========================
+Access Data From a Cursor
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: read, results, code example
+
+Overview
+--------
+
+In this guide, you can learn how to access data from a **cursor** by using the
+{+php-library+}.
+
+A cursor is a mechanism that returns the results of a read operation in iterable
+batches. Cursors reduce both memory consumption and network bandwidth usage by holding
+only a subset of documents at any given time rather than returning all documents at
+once.
```

Comment:
A piece of feedback I got from the Java DBX team on a Kotlin Sync version of this page I had put up for PR:

> Cursors are not really concerned with bandwidth, they reduce the number of requests while being mindful of the memory consumption

The suggestion below might be a bit long so feel free to workshop.

```suggestion
batches. Cursors reduce both memory consumption and the number of requests the driver
sends to the server by holding only a subset of documents at any given time, rather than returning all documents at once.
```